aequilibrae/paths/graph.py

```diff
@@ -630,3 +630,6 @@ def __determine_types__(self, new_type, current_type):
         else:
             raise ValueError("WRONG TYPE OR NULL VALUE")
         return def_type
+
+    def save_compressed_correspondence(self, path):
+        self.graph[["link_id", "__supernet_id__", "__compressed_id__"]].to_feather(path)
```

Comment:
Don't we need link direction as well, since the compressed graph is directional?

Reply:
saving full graph and nodes_to_indices now